Mathlib/Order/WellFoundedSet.lean

```diff
@@ -203,13 +206,21 @@ def IsWF (s : Set α) : Prop :=
 theorem isWF_empty : IsWF (∅ : Set α) :=
   wellFounded_of_isEmpty _
 
-theorem isWF_univ_iff : IsWF (univ : Set α) ↔ WellFounded ((· < ·) : α → α → Prop) := by
-  simp [IsWF, wellFoundedOn_iff]
-
 theorem IsWF.mono (h : IsWF t) (st : s ⊆ t) : IsWF s := h.subset st
 
+theorem isWF_univ_iff : IsWF (univ : Set α) ↔ WellFoundedLT α := by
+  simp [IsWF, wellFoundedOn_iff, isWellFounded_iff]
+
+theorem _root_.WellFoundedLT.isWF [h : WellFoundedLT α] (s : Set α) : s.IsWF :=
+  (Set.isWF_univ_iff.2 h).mono s.subset_univ
+
+@[deprecated WellFoundedLT.isWF (since := "2025-01-16")]
+theorem _root_.WellFounded.isWF (h : WellFounded ((· < ·) : α → α → Prop)) (s : Set α) : s.IsWF :=
+  have : WellFoundedLT α := ⟨h⟩
```

Comment:
I'm slightly confused: it looks like you're introducing and immediately deprecating this theorem?

Reply:
Ah, I read further, now I see.